config/crds.yml

```diff
@@ -734,6 +734,10 @@ spec:
       jsonPath: .status.deploy.startedAt
       name: Since-Deploy
       type: date
+    - description: Target Cluster name taken from the kubeconfig.
+      jsonPath: .spec.cluster.kubeconfigSecretRef.name
+      name: TargetCluster
```

Comment:
```suggestion
      name: Target-Cluster
```

Reply:
Default ideally is the name of "this" cluster. i.e the one it's running on.

Reply:
^ Makes sense 💯 , it would be good to have the target-cluster populated with the current cluster by default, but, is `.spec.cluster.kubeconfigSecretRef.name` populated by default?

Reply:
This would require some code changes or it's about adding something like `// +kubebuilder:default:="this"` ?

Reply:
I imagine it would involve adding a `cluster` field to the `status` which is populated on reconcile. I do not imagine the controller populates any of the `spec` fields 🤔

Reply:
I definitely agree with Neil in regards to what should the default value be though!